src/main/java/mikeshafter/iciwi/faregate/FareGateListener.java

```diff
@@ -28,9 +29,11 @@
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
+import java.util.Objects;
 import java.util.regex.Pattern;
 
 import static mikeshafter.iciwi.util.MachineUtil.any;
+import static mikeshafter.iciwi.util.MachineUtil.parseComponent;
 import static org.bukkit.plugin.java.JavaPlugin.getPlugin;
 
 
```

Comment:
Deprecated and will be replaced by individual classes for Harlonian and Limarusian style fare gates
This is for easier addition of more fare gate styles in the future.